### ISSUE TITLE
Is the lib Node.js compatible?

### ISSUE THREAD

**Issue by ffknob** [futurepress/epub.js#1030]
Hi there

I'd just would like to ask if the lib is compatible with Node.js. 

This:

```
import { Book } from 'epubjs';

export const readEPub = (url: string): Promise<Book> => {
  return new Promise((resolve, reject) => {
    try {
      const book: Book = new Book(url, {});
      resolve(book);
    } catch (e) {
      reject(e);
    }
  });
};

export default { readEPub };
```

Results in:

`XMLHttpRequest is not defined`

Which makes sense, since we are not in the brwoser context.

So, what should I do in order to use the lib in the backend with Node.js?

Thank you!

**Comment by kumar-muthu** [comment]
Try using a headless browser preferably https://github.com/puppeteer/puppeteer and render your epub there

**Comment by mcthulhu** [comment]
Yes, I'm using it in a Node.js (Electron) desktop application; see https://github.com/mcthulhu/jorkens. It took a bit of trial and error to get it working in Electron, but it seems to be running well now and I'm starting to add functionality.
